pyccel/ast/internals.py

```diff
@@ -245,6 +245,103 @@ def code(self):
         """
         return self._code
 
+#------------------------------------------------------------------------------
+class CStringExpression(Basic):
+    """
+    Internal class used to hold a C string that has LiteralStrings and C macros
```

Comment:
Please add Parameters to this docstring. E.g. what types are acceptable for `args`?